Sources/Array.swift

```diff
@@ -213,6 +213,26 @@ public extension Array {
   
 }
 
+extension Array where Element: Hashable {
+  
+    /// Creates an object composed from arrays of keys and values.
+    ///
+    /// :param values The array of values.
+    /// :return Dictionary based on the keys and values passed in order.
+    public func zipObject<T>(values: [T]) -> [Element:T] {
```

Comment:
Can you use the Dollar zipObject method instead https://github.com/ankurp/Dollar/blob/master/Sources/Dollar.swift#L1308

```
public func zipObject<T>(values: [T]) -> [Element:T] {
    return $.zipObject(self, values)
}
```